Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGalleryViewModel.swift

```diff
@@ -175,6 +177,13 @@ private extension BasemapGalleryViewModel {
                 try await portal.load()
                 
                 let basemaps: [Basemap]
+                
+                if geoModel is Scene {
+                    try await portal.basemaps3D.forEach { basemap in
+                        items.append(BasemapGalleryItem(basemap: basemap))
+                    }
```

Comment:
Instead of appending one at a time, you can lazily map over the basemaps and then append that lazy sequence.

```suggestion
                    let basemaps3D = try await portal.basemaps3D
                    items.append(
                        contentsOf: basemaps3D.lazy.map { BasemapGalleryItem(basemap: $0) }
                    )
```